frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -94,10 +100,11 @@ const isPersonPropertyShortcutSearchParams = (
 }
 
 const isReplayURLSearchParams = (x: ReplayURLSearchParamTypes): x is ReplayURLSearchParams => {
+    const replayURLSearchParams = x as ReplayURLSearchParams
     return (
-        (x as ReplayURLSearchParams).filters !== undefined ||
-        (x as ReplayURLSearchParams).order !== undefined ||
-        (x as ReplayURLSearchParams).order_direction !== undefined
+        isValidRecordingFilters(replayURLSearchParams.filters) ||
+        isValidRecordingOrder(replayURLSearchParams.order) ||
+        isValidRecordingOrderDirection(replayURLSearchParams.order_direction)
     )
 }
```

Comment:
we only checked there was a value, not that there was a valid value for this logic

and since this logic is mounted everywhere, any scene setting order would trigger this